pages/search.tsx

```diff
@@ -11,13 +28,148 @@ export const getServerSideProps: GetServerSideProps = async ({ query }) => ({
   },
 })
 
-export const Search: FC<{
-  query: ReturnType<typeof mapRawQueryToState>
-}> = ({ query }) => (
-  <div>
-    <InternalLink href="/">Home</InternalLink>
-    <pre>{JSON.stringify(query, null, 2)}</pre>
-  </div>
+const SearchSuggestionItem: FC<SearchSuggestionItemPropType> = ({
+  properties,
+  onClick,
+}) => (
+  <button
+    className={classNames(
+      'flex gap-4 items-center pt-3 text-left w-full',
+      'hover:bg-gray-100 rounded transition px-4 -ml-4',
+      'group focus:outline-none focus:ring-2 focus:ring-gray-800',
+      'focus:ring-gray-800 focus:z-10 relative'
+    )}
+    style={{ width: 'calc(100% + 32px)' }}
+    onClick={onClick}
+  >
+    <span
+      className={classNames(
+        'rounded-full w-4 h-4 border-4 border-white shadow-md',
+        'transform -translate-y-2'
+      )}
+      style={{
+        backgroundColor: POI_DATA.fillColorMap.get(properties.category),
+      }}
+    />
+    <div
+      className={classNames(
+        'pb-2 border-b border-gray-300 flex-grow border-dashed',
+        'group-hover:border-opacity-0 group-focus:border-opacity-0 transition'
+      )}
+    >
+      <h6 className="text-sm leading-4">{properties.name}</h6>
+      <div className="flex gap-x-2 flex-wrap">
+        <span className="text-sm text-gray-500">{properties.category}</span>
+        <span className="text-sm text-gray-400 italic">
+          (&thinsp;{properties.conditions}&thinsp;)
+        </span>
+      </div>
+    </div>
+  </button>
+)
+
+const SearchResultItem: FC<SearchResultItemPropType> = ({
+  name,
+  searchTerm,
+  onClick,
+}) => (
+  <button
+    onClick={onClick}
+    className={classNames(
+      'flex gap-2 items-center pt-3 text-left w-full',
+      'hover:bg-gray-100 rounded transition px-4 -ml-4',
+      'group focus:outline-none focus:ring-2 focus:ring-gray-800',
+      'focus:ring-gray-800 focus:z-10 relative'
+    )}
+    style={{ width: 'calc(100% + 32px)' }}
+  >
+    <span className="text-gray-300 transform -translate-y-2">
+      <GeoPinIcon />
+    </span>
+    <div
+      className={classNames(
+        'pb-2 border-b border-gray-300 flex-grow border-dashed',
+        'group-hover:border-opacity-0 group-focus:border-opacity-0 transition'
+      )}
+    >
+      <h6 className="text-sm leading-4">
```

Comment:
I think that somewhere around here happens what leads to this:

![Screenshot 2021-06-22 at 13 39 26](https://user-images.githubusercontent.com/15640196/122918733-b95a7880-d35f-11eb-82e6-b11fc77e6ce3.png)

Is there any way that we can make this case-insensitive? Or if that is too tricky, I wouldn't mind if we just remove the gray highlighting altogether.

Reply:
Good point! I think I know a way. I'll try that out! Thx!